spec/std/http/server/server_spec.cr

```diff
@@ -541,6 +541,41 @@ module HTTP
 
       server_done.should be_false
     end
+
+    describe "#close" do
+      it "closes gracefully" do
+        server = Server.new do |context|
+          context.response.flush
+          context.response.puts "foo"
+          context.response.flush
+
+          Fiber.yield
```

Comment:
There is no doc for [`Fiber.yield`](https://crystal-lang.org/api/0.27.0/Fiber.html#yield-class-method), could you please explain how it works here?

Reply:
`Fiber.yield` gives the scheduler the possibility to yield execution to a different fiber.
This was a sloppy implementation though and has been replaced since then in #6953